service/src/main/java/uk/gov/hmcts/reform/fpl/model/common/dynamic/DynamicElementParser.java

```diff
@@ -0,0 +1,17 @@
+package uk.gov.hmcts.reform.fpl.model.common.dynamic;
+
+/**
+ * Class that allows another to be converted into a {@link DynamicListElement}.
+ */
+public interface DynamicElementParser {
```

Comment:
Please consider better name. We have had the discussion about the Parser suffix, not sure really if the elements implementing the interface are Parsers. They are just classes which can be converted ot DynamicElements. Please reconsider better name with Lewis.

Reply:
changed to DynamicElementIndicator